src/Groups/matrices/form_group.jl

```diff
@@ -163,12 +163,12 @@ function invariant_alternating_forms(G::MatrixGroup{S,T}) where {S,T}
       push!(M,MM)
    end
 
-   r,K = nullspace(block_matrix(length(M),1,M))
+   r,K = nullspace(vcat(M))
 
    L = T[]
    for j in 1:r
-      B = upper_triangular_matrix(K[1:div(n*(n-1),2),j])
-      B = insert_block(zero_matrix(F,n,n),B,1,2)
+      B = zero_matrix(F,n,n)
+      _copy_matrix_into_matrix(B,1,2,upper_triangular_matrix(K[1:div(n*(n-1),2),j]))
```

Comment:
I think something like this would be more readable and more "standard" (I am not sure if the index ranges are correct, though, please check):
```suggestion
      B[1:n-1, 2:n] = upper_triangular_matrix(K[1:div(n*(n-1),2),j]))
```